translator/translate/otel/receiver/jmx/translator.go

```diff
@@ -183,6 +186,15 @@ func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 		}
 	}
 
+	if !context.CurrentContext().GetOmitHostname() && !conf.IsSet(ec2taggerprocessor.Ec2taggerKey) {
```

Comment:
Unsure why ec2 tagger was not deleting the host dimensions for jmx metrics. So I manually do not add host when ec2 tagger is added to config